2019/swift-cheatsheet/readme.md

```diff
@@ -432,19 +444,19 @@ namesArray.append("Lauren") // will add "Lauren" to the end of the namesArray
 To **insert** an item into an array:
```

Comment:
Thanks for all the below - I am stuck thinking in an old version of Swift obviously!